modules/workflows/OrcaRecoveryWorkflow/orca_recover_workflow.asl.json

```diff
@@ -13,7 +13,8 @@
           "task_config": {
             "buckets": "{$.meta.buckets}",
             "fileBucketMaps": "{$.meta.collection.files}",
-            "excludeFileTypes": "{$.meta.collection.meta.excludeFileTypes}"
+            "excludeFileTypes": "{$.meta.collection.meta.excludeFileTypes}",
+            "restoreType": "{$.meta.collection.meta.restoreType}"
```

Comment:
can you refer where in config.json and readme.md it's not matching?

Reply:
The key added to config is "orcaRestoreTypeOverride" which I do believe is a better name.

Reply:
The key added to config is "orcaRestoreTypeOverride" which I do believe is a better name.

Reply:
Now that I'm looking closer, did you mean to add this to ExtractFilepaths? Should be RequestFiles, yes?

Reply:
Now that I'm looking closer, did you mean to add this to ExtractFilepaths? Should be RequestFiles, yes?

Reply:
yes I meant to add to RequestFiles

so I added `orcaRestoreTypeOverride` in request_files/config.json but `restoreType` in the workflow.

are you suggesting I can to this for the workflow?

"orcaRestoreTypeOverride": "{$.meta.collection.meta. orcaRestoreTypeOverride}"

Reply:
I concur with @AndrewDorn  I would keep the naming convention and key names the same.
Possibly consider `orcaDefaultRecoveryTypeOverride` so that is similar to `orcaDefaultBucketOverride` for configuration naming.

Reply:
updated to `"orcaDefaultRecoveryTypeOverride": "{$.meta.collection.meta.orcaDefaultRecoveryTypeOverride}"`

Reply:
Did we mean to switch from `Restore` to `Recovery`? Haven't seen the commit yet, but comments indicate that both are in use.

Reply:
I have not pushed the commits yet but the latest one is 

orcaDefaultRecoveryTypeOverride

Reply:
I see `orca_default_restore_RETRIEVAL_type` in main.tf and `orcaDefaultRECOVERYTypeOverride` in the workflow. We now have two competing standards.

Reply:
I am open to changing it, but it should be intentional and consistent. `recovery` may be slightly better, as it is more similar to `recoverFiles`.

Reply:
thoughts @bhazuka ?

Reply:
I also see `recovery_retrieval`